deps/v8/tools/lldb_commands.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2017 the V8 project authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import lldb
+import re
+
+def jst(debugger, *args):
+  """Print the current JavaScript stack trace"""
+  target = debugger.GetSelectedTarget()
+  process = target.GetProcess()
+  thread = process.GetSelectedThread()
+  frame = thread.GetSelectedFrame()
+  frame.EvaluateExpression("_v8_internal_Print_StackTrace();")
```

Comment:
On my system this needs to be `frame.EvaluateExpression("(void)_v8_internal_Print_StackTrace();")`
If you run it manually lldb says the expression needs a return type:
```
(lldb) p _v8_internal_Print_StackTrace()
error: '_v8_internal_Print_StackTrace' has unknown return type; cast the call to its declared return type
```